Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+![Set Value Changed Function](assets/images/SetValueChangedFunction.png)
+
+> Play the Unity scene you should now be able to control the GameObject using the A, S, D, W keys.
+
+![Move Capsule](assets/images/MoveCapsule.png)
+
+### Step 11
+
+We are now going to make it possible to rotate the capsule instead. 
+
+Disable the `Movement` GameObject.
+
+![Disable Movement GameObject](assets/images/DisableMovementGameObject.png)
+
+### Step 12
+
+Create a new `Empty` Unity Object by selecting `Main Menu -> GameObject -> Create Empty` and rename it `Rotation`. Click `Add Component` and add a `Float To Vector3` component.
+
+![Add Float To Vector3](assets/images/AddFloatToVector3.png)
+
+Click `Add Component` again and add a `Transform Euler Rotation Mutator` component.
+
+![Add Euler Rotator](assets/images/AddEulerRotator.png)
+
+### Step 13
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Euler Rotation Mutator` Facade. On the `Mutate on X` property turn off the `X` and `Z`  values.
+
+![Drag Capsule Into Euler Mutator](assets/images/DragCapsuleIntoEulerMutator.png)
+
+### Step 14
+
+Select the `Rotation` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Transformed` event parameter on the `Float To Vector3` component.
+
+Drag and drop the `Rotation` GameObject into the event listener box that appears on the `Transformed` event parameter on the `Float To Vector3` component that displays `None (Object)`.
+
+![Drag Rotation Into Event Listener](assets/images/DragRotationIntoEventListener.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `TransformEulerRotationMutator` -> `DoSetProperty` function (be sure to select `Dynamic Vector3 - DoSetProperty` for this example).
+
+![Set Transformed Function](assets/images/SetTransformedFunction.png)
+
+### Step 15
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToAngle` prefab into the Unity hierarchy window.
+
+![Import Axes To Angle](assets/images/ImportAxesToAngle.png)
+
+### Step 16
+
+On the `Input.CombinedActions.AxesToAngle` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Horizontal` GameObject into the `Horizontal Axis` property on the `Axis Settings` component.
+
+![Drag And Drop Axes To Angle Into Horizontal Axis](assets/images/DragAndDropAxesToAngleIntoHorizontalAxis.png)
+
+Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Vertical` Axis` property on the `Axis Settings` component.
```

Comment:
backticks are wrong here causing the formatting to be bad: plus it's not `Axis Settings`.

This should be formatted and worded like:

```
Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Vertical Axis` property on the `Axis To Angle Action` component.
```

Reply:
This still needs actioning.